github/ci/prow/files/config.yaml

```diff
@@ -48,8 +48,9 @@ deck:
         - podinfo.json
   tide_update_period: 1s
   rerun_auth_config:
```

Comment:
it should be `rerun_auth_configs` here: https://github.com/kubernetes/test-infra/pull/16957/commits/42d4861750244594aff8e14d57be8392bf03b119

Reply:
Ah, nice. Fixed.